ring-core/test/ring/assets/foo.bar

```diff
@@ -0,0 +1 @@
+[:div "I'm a blog"]
```

Comment:
You can remove this file. There are plenty of other generic test files that can be used instead.